extensions/mssql/src/telemetry.ts

```diff
@@ -114,6 +116,21 @@ export class Telemetry {
 		}
 
 	}
+
+	/**
+	 * Collects server information from ServerInfo to put into a
+	 * property bag
+	 */
+	public static fillServerInfo(telemetryInfo: { [key: string]: string }, serverInfo: ServerInfo): { [key: string]: string } {
+		telemetryInfo['osVersion'] = serverInfo?.osVersion;
```

Comment:
I would think this is the client OS, are we automatically adding the client information in ADS core when sending telemetry? if not, I think we should also add client os type here. the server OS version doesn't seem to be something we care about.

Reply:
This is the server OS. I don't think we get client machine OS in a connection.

Reply:
as we discussed offline, please remove this property